apps/web-mzima-client/src/app/shared/components/sidebar/sidebar.component.html

```diff
@@ -42,7 +47,13 @@
               [attr.data-onboard-id]="'sidebar-btn-' + item.icon"
               (click)="item.action && item.action()"
             >
-              <mat-icon class="sidebar-menu-button__icon" [svgIcon]="item.icon"></mat-icon>
+              <!-- Added aria-hidden true (I noticed all mat-icons in the side nav the aria-hidden is set to true in default 
+              (or perhaps there's a configuration I missed)) -->
```

Comment:
No reason to add `aria-hidden="true"` it is added by deafult you are right https://material.angular.io/components/icon/overview#accessibility